ext/lib/gui/lvgl/lv_conf.h

```diff
@@ -0,0 +1,350 @@
+/**
+ * @file lv_conf.h
+ *
+ */
+
+#ifndef LV_CONF_H
+#define LV_CONF_H
+
+/*----------------
+ * Dynamic memory
+ *----------------*/
+
+/* Memory size which will be used by the library
+ * to store the graphical objects and other data */
+#define LV_MEM_CUSTOM      1                /*1: use custom malloc/free, 0: use the built-in lv_mem_alloc/lv_mem_free*/
+#if LV_MEM_CUSTOM == 0
+#define LV_MEM_SIZE    (32U * 1024U)        /*Size memory used by `lv_mem_alloc` in bytes (>= 2kB)*/
+#define LV_MEM_ATTR                         /*Complier prefix for big array declaration*/
+#define LV_MEM_AUTO_DEFRAG  1               /*Automatically defrag on free*/
+#else       /*LV_MEM_CUSTOM*/
+
+#ifdef CONFIG_LVGL_MEM_POOL_HEAP_KERNEL
+#define LV_MEM_CUSTOM_INCLUDE <kernel.h>     /*Header for the dynamic memory function*/
+#define LV_MEM_CUSTOM_ALLOC   k_malloc       /*Wrapper to malloc*/
+#define LV_MEM_CUSTOM_FREE    k_free         /*Wrapper to free*/
+#elif defined(CONFIG_LVGL_MEM_POOL_HEAP_LIB_C)
+#define LV_MEM_CUSTOM_INCLUDE <stdlib.h>   /*Header for the dynamic memory function*/
+#define LV_MEM_CUSTOM_ALLOC   malloc       /*Wrapper to malloc*/
+#define LV_MEM_CUSTOM_FREE    free         /*Wrapper to free*/
+#else
+#define LV_MEM_CUSTOM_INCLUDE <gui/lvgl_mem.h>  /*Header for the dynamic memory function*/
+#define LV_MEM_CUSTOM_ALLOC   lvgl_malloc       /*Wrapper to malloc*/
+#define LV_MEM_CUSTOM_FREE    lvgl_free         /*Wrapper to free*/
+#endif
+
+#endif     /*LV_MEM_CUSTOM*/
+
+/*===================
+   Graphical settings
+ *===================*/
+
+/* Horizontal and vertical resolution of the library.*/
+#define LV_HOR_RES          CONFIG_LVGL_HOR_RES
+#define LV_VER_RES          CONFIG_LVGL_VER_RES
+#define LV_DPI              CONFIG_LVGL_DPI
+
+/* Size of VDB (Virtual Display Buffer: the internal graphics buffer).
+ * Required for buffered drawing, opacity and anti-aliasing
+ * VDB makes the double buffering, you don't need to deal with it!
+ * Typical size: ~1/10 screen */
+#define LV_VDB_SIZE         CONFIG_LVGL_VDB_SIZE  /*Size of VDB in pixel count (1/10 screen size is good for first)*/
```

Comment:
Could we define this as percentage of display?
```
#define LV_VDB_SIZE (CONFIG_LVGL_VDB_SIZE  * CONFIG_LVGL_HOR_RES * CONFIG_LVGL_VER_RES / 100)
```
 IMHO it will be easier to use.

Reply:
good suggestion, I will update the code